CMS/translations.py

```diff
@@ -437,6 +437,10 @@
         'en': 'Filter',
         'cy': 'Hidlo'
     },
+    'filter_by': {
+        'en': 'Filter by',
+        'cy': 'Hidlo yn ôl'
```

Comment:
Personally, as a Welsh speaker, it would make more sense if I saw _**Hidlo gan**_